controllers/dnsupdate.go

```diff
@@ -513,6 +520,7 @@ func (r *GslbReconciler) configureZoneDelegation(gslb *k8gbv1beta1.Gslb) (*recon
 				return &reconcile.Result{}, err
 			}
 		}
+
 	}
 	return nil, nil
```

Comment:
Thx to refactor into  `configureZoneDelegation` factory, we are just little step from factory - strategy patterns. 

Should we rather manage `case DNSTypeNoEdgeDNS:`  as well and return `nil, error(unhandled DNS type...)` by default ?

Reply:
@kuritka done in https://github.com/AbsaOSS/k8gb/pull/217/commits/36e2c34f3a0d81a8c9ea5c05c34e5224de561cb6